examples/c-comparisons/script.jl

```diff
@@ -0,0 +1,125 @@
+# # Binary Classification with Laplace approximation
+#
+# This example demonstrates how to carry out non-conjugate Gaussian process
+# inference using the Laplace approximation.
+#
+# For a basic introduction to the functionality of this library, please refer
+# to the [User Guide](@ref).
+#
+# ## Setup
+
+using ApproximateGPs
+using LinearAlgebra
+using Distributions
+using LogExpFunctions: logistic, softplus, invsoftplus
+#using ParameterHandling
```

Comment:
Can this just be removed?

Reply:
On a separate note, did you avoid ParameterHandling just to keep the number of packages involved in the example minimal, or was there a different reason? Not a problem, just curious.

Reply:
main reason was that it came out of my demo script that I used to check things are working, and I started out not using anything fancy - but I think overall I'm also happy having different notebooks demonstrating different ways of handling parameters, so I'll leave it as is:)
```suggestion
```